extensions/binding/org.eclipse.smarthome.binding.fsinternetradio/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,26 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: FSInternetRadio Binding
+Bundle-SymbolicName: org.eclipse.smarthome.binding.fsinternetradio;singleton:=true
+Bundle-Vendor: Eclipse.org/SmartHome
+Bundle-Version: 0.8.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .
+Import-Package: 
+ org.apache.commons.httpclient,
```

Comment:
Would it be possible to refactor the code to use the Jetty client library instead? There was a discussion about the best HTTP lib here https://www.eclipse.org/forums/index.php/t/1069192/ and we decided that we want to get rid off Apache Commons HttpClient...